package.json

```diff
@@ -18,7 +18,9 @@
     "url": "https://github.com/vriad/zod/issues"
   },
   "homepage": "https://github.com/vriad/zod",
-  "dependencies": {},
+  "dependencies": {
+    "yarn": "^1.22.4"
```

Comment:
why do we need `yarn` at the deps?

Reply:
removed. I have no yarn installed in global scope so i made a mistake. thank you